### PR TITLE
Refactor containerSizingStyles and fix how _applyStyles is used

### PR DESCRIPTION
**Pull request by ntshcalleia**

### CODE REVIEW

**Review by sambua — COMMENTED** [review]
HI @ntshcalleia these what I've faced on initial testing

- (blk=2&ad=0&os=0&stk=0&flt=1&ap=1) on mobile and desktop: when only floating sets, main player container has some height. When close floating height dissapear as should.
- (blk=2&ad=0&os=0&stk=1&flt=0&ap=1) on mobile. sidebar dissapear. Without sidebar it's OK. Video attached.
- (blk=2&ap=1&ad=0&os=0&stk=1&flt=0&prk=l) set `height: 330` when main player starts on view width fit whole width. When switch to sticky and go back it fits only video container.


https://github.com/betajs/betajs-media-components/assets/5671772/ac63efd0-ffd0-4e75-976f-34f24812b435

**Comment by ntshcalleia** [comment]
Thanks for the feedback, @sambua! I pushed some fixes, when you have the time can you check again? I'm still working on the sidebar issue.

**Comment by sambua** [comment]
Hi @ntshcalleia I've pulled, and seems still has issues on floating and back. Also where we're implementing `hideplayer` state? I'm seeing you define it but it's never used.

File: src/dynamics/video_player/player/player.js [hunk 1]
```diff
@@ -614,62 +612,64 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
...
                             aspectRatio: aspectRatio
                         };
                         if (height) styles.height = isNaN(height) ? height : parseFloat(height).toFixed(2) + "px";
                         if (width) styles.width = isNaN(width) ? width : parseFloat(width).toFixed(2) + "px";
-                        containerStyles = styles;
-                        if (isFloating && !this.get("fullscreened")) {
+                        containerStyles = floatingonly ? {} : Objs.extend({}, styles);
+                        if (corner) styles.borderRadius = corner;
+                        if (isFloating) {
                             calculated = this.__calculateFloatingDimensions();
```
Comment (sambua) [inline src/dynamics/video_player/player/player.js:645]: It's not critical but will be better to predefined variables. 
`let floatingTop, floatingBottom, floatingRight, floatingLeft, floatingWidth, floatingHeight;`

**Comment by ntshcalleia** [comment]
> Hi @ntshcalleia I've pulled, and seems still has issues on floating and back. Also where we're implementing `hideplayer` state? I'm seeing you define it but it's never used.

`hideplayer` is being used on `containerSizingStyles`

**Comment by sambua** [comment]
@ntshcalleia ah I see I thought `containerSizingStyles` is depend on `hidden` attribute. 👍

**Comment by ntshcalleia** [comment]
Hi @sambua, can you try again? Should be working as expected now.

**Comment by sambua** [comment]
Hi @ntshcalleia now, works much better only one serious bug: blk=2&ad=1&os=0&stk=0&flt=1&ap=1 (floatingonly) close works and not hide player container as expected

Also, probably not related to this task, but I think it's will be better apply with this chnages related styling: .ba-commoncss-full-width when only height is set not looks good, without will be better to fit player itself only not whole container.

**Pull request merged**